Project.toml

```diff
@@ -0,0 +1,7 @@
+name = "PkgLicenses"
+uuid = "fc669557-7ec9-5e45-bca9-462afbc28879"
```

Comment:
I assume you've made sure this matches what's in the registry?

Reply:
Yes.  I copied it form there.